packages/dso-toolkit/src/styles/mixins/_typography.scss

```diff
@@ -132,3 +132,28 @@ hr {
   margin-bottom: $dso-vertical-spacing-medium;
   margin-top: $dso-vertical-spacing-medium;
 }
+
+// Components
+.banner,
+dso-banner,
+.dso-banner:not(.row) {
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    color: $zwart;
+    font-size: 1rem;
+    line-height: $line-height-base;
+    margin-bottom: 0;
+
+    + p {
+      display: inline;
+
+      + *:not(button[slot="button"]) {
```

Comment:
Waarom is deze selector nodig, en waarom is er voor de oude situatie geen uitzondering op 'n element nodig wat naast de `p` kan staan?